### PR TITLE
Setup Test Infrastructure - Using Jest

### PR DESCRIPTION
**Pull request by bhongy**
Just setting infrastructure. I will create separate PRs for actual tests.

### CODE REVIEW

**Comment by bhongy** [comment]
@supasate want me to update README related to adding tests — where, command to run, etc —  as a part of this PR?

**Comment by supasate** [comment]
Great!

Let's edit the README after creating some real test stuff. May put it as a sub-section of the `Build` section.

File: test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { LOCATION_CHANGE } from '../src/actions'
+
+describe('Example Test', () => {
+  it('should pass', () => {
+    expect(LOCATION_CHANGE).toEqual('@@router/LOCATION_CHANG')
```
Comment (supasate) [inline test/index.test.js]: @@router/LOCATION_CHANG => @@router/LOCATION_CHANGE

Comment (supasate) [inline test/index.test.js]: Just saw it after merging. I'll make a new push to fix it then.

**Comment by bhongy** [comment]
Yeah, I wanted to see the error message. It's not the real test anyway. We
can also fix it when writing real tests.

**Pull request merged**